velox/type/Timestamp.h

```diff
@@ -113,10 +113,14 @@ struct Timestamp {
     VELOX_USER_DCHECK_LE(nanos, kMaxNanos, "Timestamp nanos out of range");
   }
 
-  /// Creates a timestamp from the number of days since the Julian epoch
-  /// and the number of nanoseconds.
+  // Creates a timestamp from the number of days since the Julian epoch
+  // and the number of nanoseconds.
```

Comment:
nit: the original code is correct. We need to use '///' for multiple-line comments of public APIs.